modules/hitlnext/src/views/lite/HandoffAssigned.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Fragment, useEffect, useState } from 'react'
+
+import lang from '../lang'
+
+import style from './style.scss'
+import MessageList from './MessageList'
+
+async function initLang(): Promise<void> {
+  return new Promise(resolve => {
+    lang.init()
+    setTimeout(resolve, 100) // hack to make sure translations are initialized before 1st rendering
+  })
+}
```

Comment:
Should not be necessary

Reply:
It is ... that's for custom components injected in channel-web. Translations are not initialized